packages/commercetools/api-client/src/types/Api.ts

```diff
@@ -37,6 +37,7 @@ export interface FilterOption {
 export interface CategoryWhereSearch extends BaseSearch {
   catId?: string;
   slug?: string;
+  onlyParents?: boolean;
```

Comment:
if it is used to fetch only the root categories isn't it better to call it `rootOnly` ? Also how popular this use case is? In how many places we would like to fetch the root categories? Maybe a custom GQL query is a better idea here